data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2748,7 +2748,7 @@
 
         [message]
             speaker=Camerin
-            message= _ "That blasted skeleton! Even fire has no effect on him!"
+            message= _ "Blast it! Even fire has no effect on him!"
```

Comment:
I suppose it's the same to me either way, but I thought your commentary said this was an attempt to add more humour. I... don't see it.

Reply:
"Blast it!" It's a pun. He's using fire. Not everyone will get it, but I thought it was humorous.

Reply:
I know Camerin is the crazy fire mage. I just don't think the joke works that well here. But I'm okay with the change.

Reply:
So the jokes not funny? That's OK. It does sound more like something a mage would say in the heat of battle.
(Another bad joke)

Reply:
I think the joke's there in both the old and the new version - either someone should get it from both versions, or they'll get it from neither. With that in mind, the old version using the past tense is a better fit for Camerin having discovered that fire isn't effective.

Reply:
I don't think it's that funny either but the new one does sound a little more natural to me at least for Camerin.